pom.xml

```diff
@@ -45,12 +45,33 @@
     <module>manager</module>
     <module>executor</module>
     <module>shard-operator</module>
-    <module>shard-operator-integration-tests</module>
-    <module>integration-tests</module>
     <module>ingress</module>
     <module>infra</module>
-    <module>use-cases</module>
     <module>rhoas-client</module>
   </modules>
 
+  <profiles>
+    <profile>
+      <id>default</id>
+      <activation>
+        <property>
+          <name>default</name>
+        </property>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <modules>
```

Comment:
Can you add a comment here like `add here all the modules that should not be in the productized profile`?

Reply:
done